src/utils/datetime_utils.js

```diff
@@ -0,0 +1,25 @@
+function getDateTime(timestamp) {
```

Comment:
In a util functions file, we should be documenting each function as much as possible so other people know how to use it. Add some comments here to describe the function. Make sure to mention what arguments it takes and what it returns.